programs/marginfi/src/instructions/marginfi_account/deposit.rs

```diff
@@ -99,6 +99,7 @@ pub fn lending_account_deposit<'info>(
     )?;
 
     bank_account.deposit(I80F48::from_num(deposit_amount))?;
+    marginfi_account.last_update = bank_account.balance.last_update;
```

Comment:
nit: this syntax using the balance's state is a little harder to understand than just clock.unix_timestamp (as above in accrue interest), since the balance is updated a couple layers deeper.

Reply:
That's a good question. The numbers will still be the same though right? `balance.last_update` ought to be the `clock.unix_timestamp` since that number is fixed for the whole block. Unless there's some situation where those two number don't match because there was no update? 

Either way I disprove of adding more updates into the instruction function body.